distribution/docs/src/main/resources/_contents/_platform-contents/extending-platform-contents.adoc

```diff
@@ -43,7 +43,8 @@ These are available for use by third party developers but their versions can cha
 The Action Framework was designed as a way to limit dependencies between applications (apps) in a system.
 For instance, a feature in an app, such as a Atom feed generator, might want to include an external link as part of its feed's entries.
 That feature does not have to be coupled to a REST endpoint to work, nor does it have to depend on a specific implementation to get a link.
-In reality, the feature does not identify how the link is generated, but it does identify whether link works or does not work when retrieving the intended entry's metadata. \Instead of creating its own mechanism or adding an unrelated feature, it could use the Action Framework to query out in the OSGi container for any service that can provide a link.
+In reality, the feature does not identify how the link is generated, but it does identify whether link works or does not work when retrieving the intended entry's metadata.
+Instead of creating its own mechanism or adding an unrelated feature, it could use the Action Framework to query out in the OSGi container for any service that can provide a link.
```

Comment:
"...query out in..." sounds odd. I think "...query out to..." sounds more idiomatic.